src/nn.h

```diff
@@ -32,6 +32,17 @@ extern "C" {
 #include <errno.h>
 #include <stddef.h>
 
+/*  64-bit integer type for statistics.  */
+#if defined(_WIN32)
+typedef	 __uint64 NN_UINT64
```

Comment:
Yes, even without a compiler in front of me, this feels wonky; I think `uint64_t` is defined by `<stdint.h>` on Windows for MSVC compiler.